deliveries/vendor/vendor.js

```diff
@@ -1,25 +1,22 @@
 'use strict';
 
-const vendor = require('../../eventpool');
-require('../driver/driver')
-require('../hub')
 const faker = require('faker')
+const client = require('socket.io-client');
+const CAPS = client.connect('http://localhost:3030/caps');
 
 var customerName = faker.name.findName();
-var storeName = faker.company.companyName();
+var storeName = '1-206-flowers';
 var address = faker.address.streetAddress();
 var orderId = faker.finance.account();
 
-vendor.on('delivered', handlePickUp)
-
-
+CAPS.emit('join', storeName)
+CAPS.on('delivered', handlePickUp)
 
 function handlePickUp(payload){
     console.log('Thank you', payload.orderID)
 };
 
-
-vendor.emit('pick-up', {
+CAPS.emit('pick-up', {
```

Comment:
Wrap CAPS.emit in a setInterval( () => { 
...
}, 3000) function and you will get a new order every three seconds in this example.